addons/pos_mercado_pago/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+access_pos_terminal,access.pos.terminal,model_pos_terminal,base.group_system,1,1,1,1
```

Comment:
This means that only the manager can use pos_terminal templates, but I think the standard user will need to access them during the payment process?